scripts/clone_template.py

```diff
@@ -246,11 +279,38 @@ def main(**kwargs):
         return 10
 
     if kwargs.get('outfile') or kwargs.get('deploy'):
+        # todo: to get rid of those scripts in jenkins or develop them from scratch
         with open(kwargs['outfile'], 'w') as outfile:
-            outfile.write("appliance_ip_address={}\n".format(ip))
-
-    # In addition to the outfile, drop the ip address on stdout for easy parsing
-    print(ip)
+            if provider_type == 'openshift':
```

Comment:
This would require additional changes to CI configuration, since existing jobs/scripts are built to consume this 'appliance_ip_address' key/value file.  The intent here wasn't to write an env.yaml file from that.  Do you have matching changes for the CI use of this script?

Reply:
Of course, I created appropriate PR in cfme-data repo